en/docs/admin-portal/applications/new-application.md

```diff
@@ -0,0 +1,45 @@
+# Add new application
```

Comment:
"Register a new application" ?

Reply:
because I feel like we are more of registering a physical application here to provide SSO for that Application.